### PR TITLE
Jh/better base url

### PR DESCRIPTION
**Pull request by joelhooks**

### CODE REVIEW

File: lib/config.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const config = {
```
Comment (joelhooks) [inline lib/config.ts:1]: this config makes a ton of sense ❤️

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
   "javascript.format.insertSpaceAfterOpeningAndBeforeClosingNonemptyBraces": false,
   "javascript.preferences.quoteStyle": "single",
-  "typescript.tsdk": "node_modules/typescript/lib",
-}
\ No newline at end of file
+  "typescript.tsdk": "node_modules/typescript/lib"
```
Comment (joelhooks) [inline .vscode/settings.json:4]: This one is nice because VS Code mismatches and it gets weird.

File: components/ReactPlayer/props.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import PropTypes from 'prop-types'
-import noop from '../../lib/noop'
+import noop from '../../utils/noop'
```
Comment (joelhooks) [inline components/ReactPlayer/props.js:2]: Util or Lib? lol

Util.

File: jest.config.js [hunk 4]
```diff
@@ -5,7 +5,12 @@ module.exports = {
     '!**/node_modules/**',
   ],
   setupFilesAfterEnv: ['<rootDir>/setupTests.js'],
-  testPathIgnorePatterns: ['/node_modules/', '/.next/', '/.vercel/'],
+  testPathIgnorePatterns: [
+    '/node_modules/',
+    '/.next/',
+    '/.vercel/',
+    '/cypress/',
```
Comment (joelhooks) [inline jest.config.js:12]: Jest was picking these up.

File: lib/search-url-builder.ts [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import config from './config'
+import {last, first} from 'lodash'
+
+export const buildTitleFromUrl = (url) => {
```
Comment (joelhooks) [inline lib/search-url-builder.ts:4]: Just an initial poke at this idea. It's a giant of statement at the end of the day feels like?

Comment (johnlindquist) [reply lib/search-url-builder.ts:4]: Was it important to keep versions of titles or only URLs? Probably wanna version maps of titles/urls then add a version arg:

`buildTitleFromUrl = (url, version) => `

Comment (joelhooks) [reply lib/search-url-builder.ts:4]: It only matters for the URLs as far as search engines are concerned. 

Titles are pretty mutable. 

The URL builder is from search filter/facet set

**Pull request merged**